R/calculate_study_day.R

```diff
@@ -0,0 +1,142 @@
+#' `calculate_study_day` performs study day calculation
+#' @description
+#' This function takes the an input data.frame and a reference data.frame (which
```

Comment:
```suggestion
#' This function takes the an input data frame and a reference data frame (which
```
I think we need to use the phrase "data frame" instead of "data.frame"